pkgs/by-name/te/textlsp/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  python3,
+  fetchPypi,
+  lib,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "textlsp";
+  version = "0.3.2";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Dn3Jw7B3IjwJLivPtGeWJT1+6VV1UysB8T4PSPaqI1U=";
+  };
```

Comment:
fetchFromGitHub or GitLab or Gitea or whever is preferred to Pypi.

Reply:
Nit: also, add a space between it and the version. Also, don't set repo to pname, have it separate.